src/components/app/SearchBar.spec.tsx

```diff
@@ -23,6 +23,27 @@ describe('SearchBar', () => {
       expect(getByRole('textbox', { name: 'Search' })).to.have.value('keyword')
     })
 
+    it('fires an event after half a second when the user types a query', async () => {
```

Comment:
That sounds kinda sluggish/non-responsive

Reply:
Ah this test could be better named! It’s a debounce, so we update the search based on the users typing, but not so we hammer the search on each keystroke.